operators/video_streaming/streaming_client_enhanced/python/tests/test_streaming_client_op_bindings.py

```
@@ -0,0 +1,255 @@
+                    fps=30,
+                    server_ip="127.0.0.1",
+                    signaling_port=48010,
+                    send_frames=False,
+                    receive_frames=False,
+                )
+                # Note: Not adding to workflow to avoid execution
+
+        app = TestApp()
+        assert app is not None
+
+    def test_operator_in_fragment(self, holoscan_modules, streaming_client_op_class, fragment):
```

Ruff (0.14.2)

Unused method argument: `holoscan_modules`

(ARG002)